hawkbit-repository/hawkbit-repository-core/src/main/java/org/eclipse/hawkbit/repository/RolloutHelper.java

```diff
@@ -68,12 +69,14 @@ public static RolloutGroup verifyRolloutGroupHasConditions(final RolloutGroup gr
      * 
      * @param amountGroup
      *            amount of groups
+     * @param maxGroups
+     *            maximum number of groups allowed
      */
-    public static void verifyRolloutGroupParameter(final int amountGroup) {
+    public static void verifyRolloutGroupParameter(final int amountGroup, final int maxGroups) {
```

Comment:
In my opinion it's better to make a real bean and inject the `QuotaManagement` here. Because now every developer can set another `maxGroups` constraint for every call.